plugins/BEdita/API/src/Error/ExceptionRenderer.php

```diff
@@ -43,9 +43,10 @@ public function render()
     {
         $isDebug = Configure::read('debug');
 
-        $code = $this->_code($this->error);
-        $message = $this->_message($this->error, $code);
+        $status = $this->_code($this->error);
```

Comment:
naming changes reflect error json output from JSON API spec http://jsonapi.org/format/#error-objects
http error code will be displayed as `status` etc etc...

see also `jsonError()` method to better understand naming choice